components/record/MediaActionBar.vue

```diff
@@ -57,6 +77,18 @@
       rightsStatement: {
         type: String,
         default: null
+      },
+      providerName: {
```

Comment:
Let's name this dataProviderName as provider and data provider are two distinct properties of records.